### PR TITLE
docs(jekyll): Add more detail about installing rbenv

### PR DESCRIPTION
**Pull request by ttomsu**
@ewiseblatt Please give these instructions a go.

### CODE REVIEW

**Review by lwander — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,9 +1,29 @@
-1. Install your favorite ruby version manager (rvm, rbenv, etc)
-	1. `rvm install ruby`, or
-	1. `rbenv install 2.4.1`
-1. `gem install bundle`
-1. `bundle install`
-1. `bundle exec jekyll serve --watch [--incremental]`
+Swap out `rbenv` below for `rvm` if you prefer. RVM was giving me installation issues, so I found `rbenv` - Travis
+
+1. Create and run from a fresh VM instance:
+  1. `gcloud compute instances create jekyll`
```
Comment (skim1420) [inline README.md]: Specify other useful parameters e.g. --image-project=ubuntu-os-cloud --image-family=ubuntu-1404-lts --machine-type=n1-standard-1

File: README.md [hunk 2]
```diff
@@ -1,9 +1,29 @@
...
+    1. `git clone https://github.com/rbenv/rbenv.git ~/.rbenv`
+    1. `git clone https://github.com/rbenv/ruby-build.git ~/.rbenv/plugins/ruby-build`
+    1. `echo 'export PATH="$HOME/.rbenv/bin:$PATH"' >> ~/.bashrc`
+    1. `echo 'eval "$(rbenv init -)"' >> ~/.bashrc`
+    1. `. ~/.bashrc`
+1. Install and use ruby 2.4.1    
+    1. `rbenv install 2.4.1`
+    1. `rbenv global 2.4.1`
+1. Checkout this repo:
+    1. `git clone https://github.com/spinnaker/spinnaker.github.io.git`
```
Comment (skim1420) [inline README.md]: Should have the user fork this repo and clone their own

**Review by skim1420 — COMMENTED** [review]
This is helpful, thanks.

File: README.md (hunk 2 above)
Comment (ttomsu) [reply README.md]: Good point. Done.

File: README.md (hunk 1 above)
Comment (ttomsu) [reply README.md]: Done.

**Comment by ttomsu** [comment]
Thanks for running through the instructions

**Pull request merged**